x-pack/plugins/data_visualizer/public/application/index_data_visualizer/hooks/use_data_visualizer_grid_data.ts

```diff
@@ -101,7 +102,10 @@ export const useDataVisualizerGridData = (
 
     if (searchData === undefined || dataVisualizerListState.searchString !== '') {
       if (dataVisualizerListState.filters) {
-        data.query.filterManager.setFilters(dataVisualizerListState.filters);
+        const globalFilters = filterManager?.getGlobalFilters();
+
+        if (filterManager) filterManager.setFilters(dataVisualizerListState.filters);
```

Comment:
When doing a browser refresh on a view which combines a saved search filter and a pinned filter, the pinned filter is lost:

![dv_pinned_filter_refresh](https://user-images.githubusercontent.com/7405507/181233338-24f6fc86-44a9-4f0a-b5fe-63f00e083c4b.gif)

Reply:
I was able to reproduce this as well, which I will address in a follow up 👍